utils/dependencies.bzl

```diff
@@ -37,6 +37,13 @@ cc_library(
     remote = "https://github.com/juloberno/diadem"
   )
 
+  _maybe(
+    git_repository,
+    name = "libtensorflow-RL-MCTS",
+    commit = "31adfc2112ce41b81574b6f44192ec47f0853fbc",
+    remote = "https://github.com/steven-guo94/libtensorflow_so"
```

Comment:
why did you set up your own repository to build libtensorflow.so?
This has the risk that the tensorflow repository and yours diverge in the future.

the tensorflowlib.so can be build using the main repo using`bazel build --config=opt //tensorflow:libtensorflow.so` right?

So why do you not use tf as a dependency here?

Reply:
see here: https://github.com/tensorflow/tensorflow/blob/0ec146da5fee12617a944127f18998acd74414d8/tensorflow/BUILD#L755

Reply:
Or if you only require the c-api:
https://github.com/tensorflow/tensorflow/blob/0ec146da5fee12617a944127f18998acd74414d8/tensorflow/c/BUILD#L143

Reply:
But we also need the .so files.  There isn't .so files in this repository. Is it ok for us to upload the .so files to bark-ml?

Reply:
For me it is fine if we have this dependency only for the model loader. But: please comment what this dependency contains, and also put a readme into the repository with the link to the tutorial for building the library and also the tf version number which was built.